src/styles/index.scss

```diff
@@ -1,3 +1,300 @@
+@import 'variables';
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
 }
+
+/* #region HEADER */
+
+@mixin flex-center($justify: space-between, $align: center) {
+  display: flex;
+  justify-content: $justify;
+  align-items: $align;
+}
+
+.header {
+  @include flex-center;
+
+  height: 60px;
+  padding: 0 50px;
+
+  &__nav {
+    padding: 0;
+
+    // Навігаційна панель
+    &-list {
+      list-style: none;
+      padding: 0;
+      margin: 0;
+
+      @include flex-center(flex-start);
+    }
+
+    &-item {
+      margin-right: $nav-item-margin;
+      font-size: $font-size-small;
+      font-weight: $font-weight-medium;
+      text-transform: uppercase;
+
+      &:last-child {
+        margin-right: 0;
+      }
+
+      &__link {
+        color: #060b35;
+        text-decoration: none;
+        display: block;
+        line-height: $line-height-nav;
+
+        &:hover,
+        &:active {
+          color: $primary-color;
+        }
+
+        &:active {
+          text-decoration: underline;
+        }
+
+        &.is-active {
+          position: relative;
+          color: $primary-color;
+
+          &::after {
+            content: '';
+            display: block;
+            width: 100%;
+            height: 4px;
+            background-color: $primary-color;
+            border-radius: 2px;
+            position: absolute;
+            bottom: 0;
+            left: 0;
+          }
+        }
+      }
+    }
+  }
+
+  &__logo-link,
+  &__logo-link img {
+    width: $logo-size;
+    height: auto;
+    display: block;
+  }
+}
+
+/* #endregion HEADER */
+
+/* #region CATALOG */
+.catalog {
+  display: grid;
+  width: 944px;
+
+  grid-template-columns: repeat(4, 200px); /* 4 колонки по 200px */
+  justify-content: center;
+  gap: 46px 48px;
+  margin: 50px auto;
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    width: 1024px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .catalog {
+    width: 1200px;
+  }
+}
+
+/* #endregion CATALOG */
+
+/* #region CARD */
+
+@mixin flex-size-center($width, $height) {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $width;
+  height: $height;
+}
+
+@mixin star-position {
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+// Міксин для стилю тексту
+@mixin text-style($font-size, $font-weight, $color) {
+  font-size: $font-size;
+  font-weight: $font-weight;
+  color: $color;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  width: $card-width;
+  height: $card-height;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:hover {
+    transform: scale(1.2);
+    transition: transform 300ms ease;
```

Comment:
The `transition` property should be placed outside of the `hover` selector. It should be in the main class selector. Otherwise, the transition will only occur when the hover state is activated, but not when it's deactivated.